.eslintrc

```diff
@@ -0,0 +1,113 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+
+  "rules": {
+    // Maybe enable?
+    "one-var": 0,
+    "vars-on-top": 0,
+    "eqeqeq": 0,
+    "yoda": [0, "never"],
+    "no-use-before-define": 0,
+    "no-mixed-requires": [0, false],
+
+    // Discuss?
```

Comment:
well, somebody has to :smile:

I didn't know about ESLint dropping the defaults.  so I'm not sure which the defaults actually are.  but... this is what I currently work with (plus defaults):

```yaml
---
env:
  node: true
rules:
  brace-style: 0
  camelcase: 0
  comma-style:
    - 2
    - last
  consistent-this:
    - 2
    - self
  consistent-return: 0
  func-style:
    - 2
    - declaration
  max-len:
    - 2
    - 80
    - 4
  max-params:
    - 1
    - 4
  no-shadow: 0
  no-cond-assign:
    - 2
    - except-parens
  no-else-return: 2
  no-floating-decimal: 2
  no-multiple-empty-lines:
    - 2
    - max: 2
  no-nested-ternary: 2
  no-reserved-keys: 2
  no-return-assign: 0
  no-self-compare: 2
  no-underscore-dangle: 2
  no-use-before-define: 0
  no-void: 2
  operator-assignment:
    - 2
    - always
  quotes:
    - 2
    - single
  radix: 2
  space-after-keywords:
    - 2
    - always
  space-before-blocks:
    - 2
    - always
  spaced-line-comment: 0
  valid-jsdoc:
    - 1
    - requireReturn: false
  vars-on-top: 2
  wrap-iife: 2
```

so this is the direction I'd like to go in.

(and yes, I'd prefer yaml)